pkg/kds/service/server.go

```diff
@@ -102,6 +109,12 @@ func (g *GlobalKDSServiceServer) streamEnvoyAdminRPC(
 	}
 	clientID := ClientID(stream.Context(), zone)
 	logger := log.WithValues("rpc", rpcName, "clientID", clientID)
+	for _, filter := range g.filters {
+		if err := filter.InterceptServerStream(stream); err != nil {
+			logger.Info("stream interceptor terminating the stream", "cause", err)
```

Comment:
why Info and not Error?

Reply:
Cause it's a user error no?

Reply:
Actually yeah feels like the error type should from the interceptor no?

Reply:
right. Changed the code to check what was the error code from interceptor. If Invalid then just do info, otherwise error